docker-compose.yml

```diff
@@ -57,6 +59,10 @@ services:
     depends_on:
       - compreface-api
       - compreface-admin
+    environment:
+      - CLIENT_MAX_BODY_SIZE=${client_max_body_size}
+    volumes:
+      - "./ui/nginx/templates:/etc/nginx/templates/"
```

Comment:
our users won't have this folder.
You need to update [dockerfile](https://github.com/exadel-inc/CompreFace/blob/master/ui/docker-prod/Dockerfile), so this template will be in docker image.